modules/workflow-service-api/src/main/java/org/opencastproject/workflow/api/WorkflowDefinition.java

```diff
@@ -143,11 +143,30 @@
    */
   String[] getTags();
 
+  /**
+   * Returns the roles for this workflow definition
+   *
+   * @return the tags
+   */
+  Collection<String> getRoles();
+
   /**
    * Removes all tags associated with this workflow definition
    */
   void clearTags();
 
+  /**
+   * Sets the organization associated with this workflow (or <code>null</code>, if it's global)
+   * @param organization the organization
+   */
+  void setOrganization(String organization);
```

Comment:
Why do we need a set function for this but not for the roles?

Reply:
Seems we don't. I have removed it.